js/main.js

```diff
@@ -100,3 +122,24 @@ function checkEmptyList(event) {
     $noEntriesText.className = 'no-entries hidden';
   }
 }
+
+$list.addEventListener('click', function (event) {
+  var dataObjectIndex = $entriesNodeList.length - 1;
+  if (event.target.matches('i')) {
+    $entriesListView.className = 'view hidden';
+    $newEntriesView.className = 'view';
+    for (var listItemIndex = 0; listItemIndex < $entriesNodeList.length; listItemIndex++) {
+      if (event.target.closest('li') === $entriesNodeList[listItemIndex]) {
+        $targetedListItem = event.target.closest('li');
+        dataObjectIndex -= listItemIndex;
```

Comment:
This is bad news, you should never use `id`s for math. Instead, you should loop over the `data.entries` list and find the entry with a matching id. This will become a big problem once you are able to delete entries.